README.md

```diff
@@ -51,9 +51,8 @@ In Netlify, this directory is the
 
 A source folder can contain:
 
-- Sub-directories with a main file called `index.js`, `index.ts`, `{dir}.js` or `{dir}.ts` where `{dir}` is the
-  sub-directory name.
-- `.js` or `.ts` files (Node.js)
+- Sub-directories with a main file called `index.js` or `{dir}.js` where `{dir}` is the sub-directory name.
```

Comment:
Did you mean to remove `index.ts` here?

Reply:
Yes, my thought was to either add all extensions, which would be veeerrrrry long, or only add `js` as an example as the next line describes which extensions are allowed.

Reply:
Sounds good! 👍🏻